test/e2e_node/cpu_manager_test.go

```diff
@@ -410,15 +415,15 @@ func runCPUManagerTests(f *framework.Framework) {
 
 		By("by deleting the pods and waiting for container removal")
 		deletePods(f, []string{pod1.Name, pod2.Name})
-		waitForContainerRemoval(fmt.Sprintf("%s_%s", pod1.Spec.Containers[0].Name, pod1.Name))
-		waitForContainerRemoval(fmt.Sprintf("%s_%s", pod2.Spec.Containers[0].Name, pod2.Name))
+		waitForContainerRemoval(pod1.Spec.Containers[0].Name, pod1.Name, pod1.Namespace)
+		waitForContainerRemoval(pod2.Spec.Containers[0].Name, pod2.Name, pod2.Namespace)
 
 		setOldKubeletConfig(f, oldCfg)
 	})
 }
 
 // Serial because the test updates kubelet configuration.
-var _ = SIGDescribe("CPU Manager [Feature:CPUManager]", func() {
+var _ = SIGDescribe("CPU Manager [Serial] [Feature:CPUManager]", func() {
```

Comment:
The `[Serial]` tag was removed as the CPU manager tests are run as a separate job in the CI/CD env. We can add it back when we want to run it as a part of the node e2e serial tests.

Reply:
@balajismaniam Actually the test is running in regular node e2e test today (Search "CPU Manager" in https://storage.googleapis.com/kubernetes-jenkins/logs/ci-kubernetes-node-kubelet/9809/build-log.txt), it is just skipped because the node has less than 2 CPUs.
Given that I prefer run and skip this in Serial test suite.

Reply:
Alright. Sounds good.